projects/observability/src/shared/dashboard/widgets/metric-display/metric-display-widget.model.ts

```diff
@@ -66,7 +71,18 @@ export class MetricDisplayWidgetModel {
   public api!: ModelApi;
 
   public getData(): Observable<MetricWidgetValueData> {
-    return this.api.getData<unknown>().pipe(mergeMap(receivedValue => this.normalizeData(receivedValue)));
+    return this.api.getData<unknown>().pipe(
+      // tslint:disable-next-line: no-any
+      mergeMap((receivedValue: any) => {
+        if (typeof receivedValue === 'object' && receivedValue.value === undefined) {
```

Comment:
Can we not do this part in the `getData` handler? So that the value that we receive here has `value` and `name` defined already?

Reply:
Didn't get it. Where else would we do it?

For other screens, the `receivedValue` contains 3 values including a key for `value`.

<img width="544" alt="Screenshot 2022-12-13 at 2 36 36 PM" src="https://user-images.githubusercontent.com/29686866/207274868-b2740012-4e20-4867-ba4d-ac4aafad0ae5.png">

For explorer screen, `receivedValue` is instead an object containing the values of the multiple metrics.

<img width="443" alt="Screenshot 2022-12-13 at 2 39 21 PM" src="https://user-images.githubusercontent.com/29686866/207275251-8ff649e5-0232-4457-9ff3-b9724cab34f3.png">

The return statement uses the title of the widget to figure out which key in this object to use.